packages/logger/src/Logger.ts

```diff
@@ -148,16 +149,16 @@ class Logger implements ClassThatLogs {
 
   private addToPowertoolLogData(...attributesArray: Array<Partial<PowertoolLogData>>): void {
     attributesArray.forEach((attributes: Partial<PowertoolLogData>) => {
-      this.powertoolLogData = merge(this.getPowertoolLogData(), attributes);
+      this.powertoolLogData = merge(attributes, this.getPowertoolLogData());
```

Comment:
(just curious) I understand that the order of param doesn't matter here. What's the reason to swap?

Reply:
See original PR comment:
> Updated calls for merge in main Logger class since it was using the lodash/fp version [that swaps the argument order](https://www.measurethat.net/Benchmarks/Show/7660/0/lodash-merge-vs-deepmerge-vs-lodashfp-merge).

But the TL;DR; is:
- Original version was using `import { merge } from 'lodash/fp';` this version of `lodash` [behaves differently than regular `lodash`](https://dev.to/ifarmgolems/what-is-lodash-fp-even-4ohd).
- As such, using the `import merge from `lodash.merge`;` (that we are using in this PR and is the _non /fp_ version) without changing the args order would make unit tests fail (because the logs objects were merged still recursively but in different order, so with different end result).
- After some digging I found that these two implementations are similar but accept params in different order ([see original link](https://www.measurethat.net/Benchmarks/Show/7660/0/lodash-merge-vs-deepmerge-vs-lodashfp-merge)).
- After swapping the order unit tests were passing again.

With that said, I really never use `lodash` so if anyone spots any issue or I have misunderstood how they behave I'll be more than willing to revert.

Reply:
Thanks, thi

> With that said, I really never use lodash so if anyone spots any issue or I have misunderstood how they behave I'll be more than willing to revert. 

That makes two of us. I only used it to learn FP many years ago.  
Don't worry. We can pin this on Sara if this breaks :D